core/Db/Schema/Mysql.php

```diff
@@ -153,8 +153,7 @@ public function getTablesCreateSql()
                               config_id BINARY(8) NOT NULL,
                               location_ip VARBINARY(16) NOT NULL,
                                 PRIMARY KEY(idvisit),
-                                INDEX index_idsite_config_datetime (idsite, config_id, visit_last_action_time),
-                                INDEX index_idsite_datetime (idsite, visit_last_action_time),
+                                INDEX index_idsite_configid_datetime (idsite, visit_last_action_time, config_id),
```

Comment:
let's rename the INDEX `index_idsite_datetime_configid` to follow index field order